apps/api/src/domain/domainModelValidators/__tests__/contextValidators/edgeConnectionValidator/buildEdgeConnectionValidatorTestCases/index.ts

```diff
@@ -281,22 +281,14 @@ export default (): EdgeConnectionValidatorTestCase[] => [
                         },
                     ],
                 },
-                expectedError: buildTopLevelError([
-                    new InvalidChronologicallyOrderedTimeRangeError({
-                        inPoint: 1200,
-                        outPoint: 1000,
-                    }),
-                ]),
+                /**
```

Comment:
Does the test validate any innerErrors. Should it have a `should ValidateInnerErrors` option insead?